### ISSUE TITLE
Choice - original, regularisation

### ISSUE THREAD

**Issue by strill66** [SASDigitalHumanitiesTraining/TextEncoding#11]
We're planning a side-by-side semi-diplomatic transcription of our text with a facing modernisation. If we use <choice> to mark up both <orig> and <reg> the xml document, does this allow for them to be differentiated between for display?

**Comment by cmohge1** [comment]
Yes, you could nest original and modernised words within a `<choice>` tag containing `<orig>` and `<reg>` for original and regularised spellings, respectively. 

E.g.

```
<choice>
<orig>voutsaf't</orig>
<reg>vouchsafed</reg>
```

Creating a facing page diplo + modernisation would be done at the processing stage -- you would basically indicate in the stylesheet to only print `<orig>`s in the diplo, and `<reg>`s in the modernised text.
